### DISCUSSION TITLE
Cypress Warning: `cy.state('withinSubject')` has been deprecated

### DISCUSSION THREAD

**Discussion by santanakrishnan** [cypress-io/cypress#26159] [General]
What is this warning message about. What command is it referring to. Please help me understanding this.

**"Cypress Warning: `cy.state('withinSubject')` has been deprecated and will be removed in a future release. You should read `cy.state('withinSubjectChain')` once at the top of your command / query, and resolve it into a value with `cy.getSubjectFromChain(withinSubjectChain)` as needed."**

**Comment by ZachMeditates** [comment]
I am curious about this as well. Does anyone have any additional information?

**Comment by gkitzing** [comment]
I would also like to know.

**Comment by crfrolik** [comment]
If you use the `cypress-pipe` plugin, it could be coming from that.
https://github.com/NicholasBoll/cypress-pipe/issues/37

**Comment by jordanpowell88** [comment]
it is related to `cy.state()` which was deprecated in release `10.5.0`. You can see the CHANGELOG entry [here](https://docs.cypress.io/guides/references/changelog#10-5-0)

**Comment by santanakrishnan** [comment]
But i am nowhere using cy.state(), then why warning is shown?

**Reply by crfrolik** [reply]
You may have installed a plugin that uses it.
